### ISSUE TITLE
Possibility to remove Appveyor

### ISSUE THREAD

**Issue by albinahlback** [wbhart/flint2#1078]
I don't like it. Could that be replaced by adding another workflow with `runs-on: windows-latest`? I know 32 bit won't be included, but perhaps one can set `ABI=32` in that case.

**Comment by wbhart** [comment]
Yes, it could definitely be replaced with GitHub actions. If you want to work on it, please do.

**Comment by wbhart** [comment]
I'll close this for now, as it proved to be infeasible.
